src/main/java/godaddy/registry/jtoolkit2/se/fee/DomainCheckFeeResponseExtension.java

```diff
@@ -109,7 +109,10 @@ private void parseFeeNodes(XMLDocument xmlDoc, FeeCheckData feeCheckData, String
 
     private FeeCheckData.Fee parseFee(Node feeNode) {
         BigDecimal feeValue = new BigDecimal(feeNode.getTextContent());
-        String description = feeNode.getAttributes().getNamedItem("description").getTextContent();
+        String description = null;
+        if (feeNode.getAttributes().getNamedItem("description") != null) {
+            description = feeNode.getAttributes().getNamedItem("description").getTextContent();
+        }
```

Comment:
Would be better doing this way:
```suggestion
        Node descriptionNode = feeNode.getAttributes().getNamedItem("description");
        if (descriptionNode != null) {
            description = descriptionNode.getTextContent();
        }
```

Reply:
Done